### ISSUE TITLE
Bug in promisified-glob/globSingleFile

### ISSUE THREAD

**Issue by christav** [Microsoft/mobile-center-cli#68]
The type signature for this code is:

```typescript
function globSingleFile(pattern: string, options?: g.Options): Promise<string>
```

The code that resolves the final promise is:
```javascript
    resolve(matches);
```

However, `matches` is an array. So the actual value returned is a `string[]`, not a `string`, which will cause all sorts of weird errors down the line.

Change that line to `resolve(matches[0]);`

**Comment by kmiecikt** [comment]
Fixed in PR #94

**Comment by christav** [comment]
Fixed.
